src/devices/src/virtio/vsock/csm/connection.rs

```diff
@@ -154,6 +155,9 @@ where
         // source, destination, credit, etc).
         self.init_pkt(pkt);
 
+        METRICS.vsock.rx_packets_count.inc();
+        METRICS.vsock.rx_bytes_count.add(pkt.len() as usize);
```

Comment:
Just asking, why do we update `rx_bytes_count` here and not when we read data from the stream (with `read_cnt` value)?

Reply:
I've added it there just for consistency, but moved it now to where you suggested.